pkg/extension/workspace/fs/interfaces.go

```diff
@@ -20,8 +20,8 @@ const (
 	extensionMetaDir = ".getenvoy/extension"
 )
 
-// WorkspaceDir represents a directory with an extension created by getenvoy toolkit.
-type WorkspaceDir interface {
+// ExtensionDir represents a directory with an extension created by getenvoy toolkit.
+type ExtensionDir interface {
```

Comment:
the same name appears in GlobalOpts which makes me confused though not sure what is better 😓

Reply:
if you look carefully, you'll notice a lot of code that is still something to clean up, but practically speaking it is hard to do it all in one PR as even this was 6 full days of work. For now, at least keeping the same name for the same thing helps as future cleanups are easier to identify. Make sense?

Reply:
sg